src/index.js

```diff
@@ -6,9 +6,7 @@ import store from './redux/store';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <Provider store={store}>
-      <App />
-    </Provider>
-  </React.StrictMode>,
```

Comment:
It would be nice if you put back the <React.strictMode></React.strictMode> for proper check on your React APP.